### ISSUE TITLE
Usersnap Feedback - Ugh bullet points copy over as images from word docs which makes for a lot [...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#897]
**Sender**: scottowenfletcher@gmail.com
**Comment**: Ugh bullet points copy over as images from word docs which makes for a lot of tedious cleanup
[Open #592 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/592)

<a href='https://usersnappublic.s3.amazonaws.com/2019-10-14/20-38/8eaff4b3-5509-4642-a180-0c98f869e422.png'>![Usersnap Feedback - Ugh bullet points copy over as images from word docs which makes for a lot (...)](https://usersnappublic.s3.amazonaws.com/2019-10-14/20-38/29563977-cd12-467e-ad8f-7a65097e7198.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2019-10-14/20-38/8eaff4b3-5509-4642-a180-0c98f869e422.png'>Download original image</a>

**Browser**: Chrome 77 (Windows 10)
**Referer**: [https://participedia.net/case/new?full=1](https://participedia.net/case/new?full=1)
**Screen size**: 1280 x 720 **Browser size**: 1920 x 794

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by jesicarson** [comment]
@scottofletcher from ALL word docs? Has this happened before? Thx.

**Comment by scottofletcher** [comment]
yup, I created a usersnap about this same issue a while back, but it must have got lost in the shuffle. I don't think you can copy-and-paste ordered or unordered lists from any word processor (Word, G Docs, etc.). this has created quite a large problem for us editors (which, I must admit, I've been reluctant to tackle) in cases that use a lot of ol/ul's (eg. basically all of the Sciencewise entries - [example](https://participedia.net/case/5672), [example](https://participedia.net/case/5705), [example](https://participedia.net/case/5670))

**Comment by jesicarson** [comment]
Darn that is annoying. It's tough to get a text editor to behave as perfectly as we'd all like when copy/pasting, unfortunately.
I will have to label this as part of our design backlog, which I am due to comb over and re-prioritize at some point once we have our current priorities budgeted for.  Thanks for hanging in there!

**Comment by scottofletcher** [comment]
ok no worries! I usually get around it by un-bulleting lists in Word before copying them in to the PPedia narrative. unfortunately not very many other contributors seem to do that (or care enough to fix their lists...)
